proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -61,6 +62,25 @@ message Resource {
 enum EventType {
   EVENT_TYPE_UNSPECIFIED = 0;
   EVENT_TYPE_ORGANIZATION_CREATED = 1;
+  EVENT_TYPE_ORGANIZATION_DELETED = 2;
+  EVENT_TYPE_ORGANIZATION_MEMBER_ADDED = 3;
+  EVENT_TYPE_ORGANIZATION_MEMBER_ROLE_CHANGED = 4;
+  EVENT_TYPE_ORGANIZATION_MEMBER_REMOVED = 5;
+  EVENT_TYPE_REPOSITORY_CREATED = 6;
+  EVENT_TYPE_REPOSITORY_DELETED = 7;
+  EVENT_TYPE_REPOSITORY_PUSHED = 8;
+  EVENT_TYPE_REPOSITORY_MEMBER_ADDED = 9;
+  EVENT_TYPE_REPOSITORY_MEMBER_ROLE_CHANGED = 10;
+  EVENT_TYPE_REPOSITORY_MEMBER_REMOVED = 11;
+  EVENT_TYPE_REPOSITORY_VISIBILITY_CHANGED = 12;
+  EVENT_TYPE_PLUGIN_CREATED = 13;
+  EVENT_TYPE_PLUGIN_DELETED = 14;
+  EVENT_TYPE_PLUGIN_PUSHED = 15;
+  EVENT_TYPE_USER_REGISTERED = 16;
+  EVENT_TYPE_USER_DELETED = 17;
+  EVENT_TYPE_USER_DEACTIVATED = 18;
```

Comment:
split out deleted / deactivated. I guess this could just be a boolean on one of the events, i.e. `is_deleted` on `message PayloadUserDeactivated`, but figured this was fine. happy to change it.